### ISSUE TITLE
bug on get main activity or other component 

### ISSUE THREAD

**Issue by lorinwu** [androguard/androguard#173]
Hi all,
I found some malicious apps register several main activities as below:
there are 2 main activities but one of them has android:enabled="false"
that will lead to get wrong main activity by androguard(a.get_main_activity method);

`<activity android:name="com.onegravity.k10.activity.DualPaneActivity" android:uiOptions="splitActionBarWhenNarrow">
            <intent-filter>
                <action android:name="android.intent.action.MAIN"/>
                <category android:name="android.intent.category.DEFAULT"/>
                <category android:name="android.intent.category.LAUNCHER"/>
                <category android:name="android.intent.category.MULTIWINDOW_LAUNCHER"/>
            </intent-filter>
        </activity>
        <activity android:enabled="false" android:icon="@drawable/ic_launcher_pro" android:label="@string/app_name_upgraded" android:name="com.onegravity.k10.activity.DualPaneActivityUpgraded" android:uiOptions="splitActionBarWhenNarrow">
            <intent-filter>
                <action android:name="android.intent.action.MAIN"/>
                <category android:name="android.intent.category.DEFAULT"/>
                <category android:name="android.intent.category.LAUNCHER"/>
                <category android:name="android.intent.category.MULTIWINDOW_LAUNCHER"/>
            </intent-filter>
        </activity>`

suggestion:
`    def get_main_activity(self):
        x = set()
        y = set()

```
    for i in self.xml:
        for item in self.xml[i].getElementsByTagName("activity"):
            activityEnabled = item.getAttributeNS(NS_ANDROID_URI, "enabled")
            if activityEnabled is not None and activityEnabled != "" and activityEnabled == "false":
                continue
            for sitem in item.getElementsByTagName( "action" ):
                val = sitem.getAttributeNS(NS_ANDROID_URI, "name" )
                if val == "android.intent.action.MAIN":
                    x.add( item.getAttributeNS(NS_ANDROID_URI, "name" ) )`
```

add 'activityEnabled = item.getAttributeNS(NS_ANDROID_URI, "enabled")' to exclude fa

**Comment by reox** [comment]
thanks for providing the bug report and also the patch!
Can you provide also sample hashes for testing?

**Comment by reox** [comment]
PR available: #174

**Comment by reox** [comment]
I think I found the app you mentioned: SHA256 cd260a0f59d6dc31475e90fb5e47f0d50305e66617c2146c1dc5240758ab5671

Looks good to me.
